stix_shifter_modules/msatp/stix_translation/README.md

```diff
@@ -1,5 +1,557 @@
 # Microsoft Defender for Endpoint (previously Microsoft Defender ATP) UDS Connector
 
+# MSATP - microsoft defender for endpoint module
```

Comment:
`Microsoft Defender for Endpoint module`

Reply:
done